### ISSUE TITLE
NTR: Adenylate kinase low in RBC (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8226]
New term request Adenylate kinase low in RBC (elthat_130419184131): Red blood cells, erythrocytes

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000558
- label: Reduced erythrocyte adenylate kinase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:7947281
- def: Activity or concentration of in the level of adenylate kinase (EC 2.7.4.3) in erythrocytes below the lower limit of normal. Adenylate kinase is a ubiquitous monomeric enzyme that catalyzes the reversible conversion of MgATP plus AMP to MgADP plus ADP and contributes to homeostasis of the adenine nucleotide composition in the cell.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8226
- synonyms:
